src/libraries/System.DirectoryServices.Protocols/tests/LdapSessionOptionsTests.cs

```diff
@@ -46,7 +47,7 @@ public void ReferralChasing_GetSetWhenDisposed_ThrowsObjectDisposedException()
             Assert.Throws<ObjectDisposedException>(() => connection.SessionOptions.ReferralChasing = ReferralChasingOptions.All);
         }
 
-        [Theory]
+        [ConditionalTheory(nameof(PlatformDetection.IsWindows))]
```

Comment:
Why ConditionalTheory instead of `[PlatformSpecific(TestPlatforms.Windows)]`?

Reply:
Is there some advantage of one over the other? I actually wasn't aware of PlatformSpecific which is why I just used ConditionalTheory and ConditionalFact

Reply:
Usually if it's just about the OS family we use PlatformSpecific, and they get pre-filtered out of execution in an early phase.  Conditional is usually used for complex conditions that can only be determined by running code, like "try loading this library, did it work?" or minimum OS versions, etc.

Failed conditionals print on the console that they didn't run.  PlatformSpecific doesn't.

Reply:
Makes sense, Fixed.